### PR TITLE
Cisco secure email added readme detailed file.

### PR DESCRIPTION
**Pull request by thangaraj-ramesh**
README_detailed file contains sample STIX patterns for Cisco Secure Email connector.

### CODE REVIEW

File: stix_shifter_modules/cisco_secure_email/README.md [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
 **Table of Contents**
 
-- [Cisco Secure Email API Endpoints](#Cisco Secure Email-api-endpoints)
+- [Cisco Secure Email API Endpoints](#cisco-secure-email-api-endpoints)
 - [Pattern expression with STIX and CUSTOM attributes - Single Observation](#single-observation)
 - [Pattern expression with STIX and CUSTOM attributes - Multiple Observation](#multiple-observation)
 - [STIX Execute Query](#stix-execute-query)
 - [Limitations](#limitations)
 - [Observations](#observations)
+- [README detailed](README_detailed.pdf)
```
Comment (mdazam1942) [inline stix_shifter_modules/cisco_secure_email/README.md:16]: Change the title to `STIX Pattern Use cases and examples`

**Review by mdazam1942 — CHANGES_REQUESTED** [review]
we can't have pdf as a doc in the repo. you need to convert the file into markdown(.md).

File: stix_shifter_modules/cisco_secure_email/README.md (hunk 1 above)
Comment (thangaraj-ramesh) [reply stix_shifter_modules/cisco_secure_email/README.md]: Changed the title as per you suggestion

**Comment by thangaraj-ramesh** [comment]
> we can't have pdf as a doc in the repo. you need to convert the file into markdown(.md).

Changed the file format to markdown(.md)

**Review by mdazam1942 — APPROVED** [review]

**Pull request merged**